spec/controllers_spec/posts_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+# spec/controllers/posts_controller_spec.rb
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
```

Comment:
- [ ] Kindly note that the tests should be of `type: :request`, not `type: :controller`, as the latter is now somewhat deprecated.